composer.json

```diff
@@ -7,11 +7,11 @@
     "require": {
         "silverstripe/recipe-plugin": "^1",
         "silverstripe/recipe-cms": "^4",
```

Comment:
```suggestion
        "silverstripe/recipe-cms": "^4.6",
```

Reply:
The other minor branches target a a specific version of recipe-cms. Not sure if it makes any difference, but I figure we might as well stay be consistent.